flake.nix

```diff
@@ -1,30 +1,207 @@
 {
   description = "Seabug";
+
   inputs = {
```

Comment:
Should we have `flake-compat` as an input?

Reply:
I noticed a kind of resentment into using it (minimize inputs number is one of the reasons) in many MLabs projects when you can reimplement what you need in like 1 line (e.g. `perSystem = nixpkgs.lib.genAttrs supportedSystems;`).
I think it's matter of taste anyway.

Reply:
@aciceri Are you referring to `flake-utils`?

Reply:
@ngua oops I misread it, yes, I meant `flake-utils`. I don't event know why `flake-compat` is needed.

Reply:
Ah I see, yeah I suggested flake-compat because `arion-pkgs.nix` reads it's version and hash from the lock file